lib/spark_api/models/listing.rb

```diff
@@ -235,6 +235,10 @@ def respond_to?(method_symbol, include_all=false)
         end
       end
 
+      def batch_photo_delete(photoIds, args={})
```

Comment:
Looking a bit more closely at the existing methods, there are bang and non-bang version of both `reorder_photo` and `reorder_photos`. It seems reasonable to follow that pattern with this method as well, with the non-bang version catching possible errors thrown by the bang version.